translations/dutch.json

```diff
@@ -131,6 +131,19 @@
 		"You are now blocking all incoming challenge requests.": "Je blokkeert nu alle inkomende uitdagingen.",
 		"You are already blocking challenges!": "Je blokkeert al uitdagingen!",
 		"You are already available for challenges!": "Je bent al beschikbaar voor uitdagingen!",
-		"You are available for challenges from now on.": "Je bent vanaf nu beschikbaar voor uitdagingen."
+		"You are available for challenges from now on.": "Je bent vanaf nu beschikbaar voor uitdagingen.",
+
+		"The announcement has ended.": "",
+		"Battles do not support announcements.": "",
+		"You are not allowed to use filtered words in announcements.": "",
+		"There is already a poll or announcement in progress in this room.": "",
+		"An announcement was started by ${user.name}.": "",
```

Comment:
Is this how translations are usually implemented?

Reply:
Looks like yes. Interesting.

Reply:
Yeah, I try to make any variables in template strings I'm going to translate descriptive enough that translators can understand with minimal technical knowledge.